src/main/java/cc/protea/util/http/Utf8Request.java

```diff
@@ -0,0 +1,305 @@
+package cc.protea.util.http;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/*
+
+ Modified with modifications copyright Richard Stanford
+
+ -- Original portions by Joe Ernst --
+
+ * Copyright 2012 Joe J. Ernst
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This class represents an HTTP Request message.
+ */
+public class Utf8Request extends Message<Utf8Request> {
```

Comment:
The class name is updated so that we can still import the `cc.protea` library while avoiding contentious class-loading cardinality issues.